stories/5-components/Links/AuLink.stories.js

```diff
@@ -82,3 +82,60 @@ Component.args = {
   width: '',
   active: false,
 };
+
+export const Secondary = Template.bind({});
+Secondary.args = {
+  text: 'Secondary',
+  route: 'index',
+  skin: 'secondary',
+  icon: 'login',
+  iconAlignment: 'left',
+  hideText: false,
+  width: '',
+  active: false,
+};
+Secondary.parameters = {
+  chromatic: { disableSnapshot: true },
+};
+
+export const PrimaryButton = Template.bind({});
+PrimaryButton.args = {
+  text: 'Primary Button',
+  route: 'index',
+  skin: 'button',
+  icon: 'login',
+  iconAlignment: 'left',
+  hideText: false,
+  width: '',
+  active: false,
+};
+
+export const SecondaryButton = Template.bind({});
+SecondaryButton.args = {
+  text: 'Secondary Button',
+  route: 'index',
+  skin: 'button-secondary',
+  icon: 'login',
+  iconAlignment: 'left',
+  hideText: false,
+  width: '',
+  active: false,
+};
+SecondaryButton.parameters = {
```

Comment:
I disabled the Chromatic tests on a few stories here because it is mostly identical anyway + more snapshots means longer runtime? If I'm wrong, make sure to correct me 🙏